server/base/src/main/java/org/apache/accumulo/server/manager/state/ZooTabletStateStore.java

```diff
@@ -134,6 +138,18 @@ public void setLocations(Collection<Assignment> assignments) throws DistributedS
 
     TabletMutator tabletMutator = ample.mutateTablet(assignment.tablet);
     tabletMutator.putLocation(assignment.server, LocationType.CURRENT);
+    if ("assignment".equals(context.getConfiguration().get(Property.TSERV_LAST_LOCATION_MODE))) {
+      TabletMetadata lastMetadata =
```

Comment:
This may not be needed in a zoo tablet state store as I believe there may be and only 1 possible last location.

Reply:
nevermind.  It appears that we are storing all cf, cq, and values now as a json blob.